.gitignore

```diff
@@ -1,14 +0,0 @@
-# IDEA files
```

Comment:
Adding a .gitignore file in your repository could be helpful, so that you don't accidentally commit files that are not needed in the source code (e.g. IntelliJ project files, .class files)